### PR TITLE
Remove image bg on dnd.su

### PR DESCRIPTION
**Pull request by BANOnotIT**
Remove background images on content blocks

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4688,6 +4688,38 @@ INVERT
 
 ================================
 
+dnd.su
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4691]: Is this the correct domain? I'm not able to visit it.

Comment (BANOnotIT) [reply src/config/dynamic-theme-fixes.config:4691]: image
Yeah. It pretty is.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -4688,6 +4688,38 @@ INVERT
...
+.input-cast_time,
+.input-style,
+.workshop__checkbox .workshop__label::before,
+#ya-site-form0 .ya-site-form__submit,
+.button,
+button:not(.lg-icon),
+input[type="submit"],
+input[type="button"] {
+   background-image: none !important;
+   background-color: var(--darkreader-neutral-background) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4718]: Is this one necessary for **all** elements listed?

Comment (BANOnotIT) [reply src/config/dynamic-theme-fixes.config:4712]: I've removed input types but yeah, most of the elements are special containers with background-image

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:4712]: I meant, if all these elements needs to have the same background-color forced. It seems like only some of them needs a new background-color.

Comment (BANOnotIT) [reply src/config/dynamic-theme-fixes.config:4712]: All these elements originally has no background-color, just background-image. The extension doesn't have any instruments to make background-image look ok. So I just replace whitish background-image with neutral background-color. If we don't set background-color all these elements become transparent.
изображение

**Review by Gusted — APPROVED** [review]

**Pull request merged**